### PR TITLE
Fix/loosen dependency version req

### PR DESCRIPTION
**Pull request by stevecox**

### CODE REVIEW

**Comment by stevecox** [comment]
@nateklaiber About half the tests still fail.  From what I can tell, it looks like creating campaigns always results in a 404.  Any tests that create a campaign in the sequence then fail.

File: setup.py [hunk 1]
```diff
@@ -15,8 +15,8 @@
     long_description_content_type="text/markdown",
     packages=find_packages(where='.', exclude=['tests.*']),
     install_requires=[
-            "requests == 2.21.0",
-            "uritemplate == 3.0.0",
-            "iso8601 == 0.1.12"
+            "requests >= 2.21.0",
```
Comment (stevecox) [inline setup.py:18]: @nateklaiber We are having conflicts with this requiring a specific version with other dependencies in our project.  This should address that.

Comment (nateklaiber) [reply setup.py:18]: @stevecox - makes sense and sounds good. I am able to use the python client against my production credentials without issues (navigating around directory, projects, campaigns, templates, etc).

**Comment by nateklaiber** [comment]
@stevecox - I am able to use the python client to interact with my production account, but the tests still have issues (i'm wondering how they ever worked). I believe it's safe to release with the updated versions, and I can address the test cleanup in a separate branch. Didn't want this to be blocking to anything, so let me know if this works.

**Comment by stevecox** [comment]
@nateklaiber Sounds good, I'll merge/release.

**Pull request merged**